pkg/cloudprovider/providers/openstack/metadata.go

```diff
@@ -36,7 +36,7 @@ const (
 	// metadataUrlTemplate allows building an OpenStack Metadata service URL.
 	// It's a hardcoded IPv4 link-local address as documented in "OpenStack Cloud
 	// Administrator Guide", chapter Compute - Networking with nova-network.
-	// https://docs.openstack.org/admin-guide/compute-networking-nova.html#metadata-service
+	// https://docs.openstack.org/nova/pike/admin/networking-nova.html#metadata-service
```

Comment:
can we use `https://docs.openstack.org/nova/latest/admin/networking-nova.html#metadata-service` i.e, `latest` instead of `pike`? thanks!

Reply:
thanks,I have fix it in pr #59029